### PR TITLE
Start stopped restart scheduled job

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/models/azure-identity/test-data.ts [hunk 1]
```diff
@@ -12,23 +12,27 @@ export const testData: TestDependencyDataType<AzureIdentityModel> = [
...
     clientId: 'any-client-id',
     serviceAccountName: undefined,
+    azureKeyVaults: undefined,
   },
   {
     __testDescription: 'Invalid empty object',
     __testIsInvalidSample: true,
     clientId: undefined,
     serviceAccountName: undefined,
+    azureKeyVaults: undefined,
```
Comment (Pespiri) [inline src/models/azure-identity/test-data.ts:36]: optional fields not required to be included in partial tests
```suggestion
  },
  {
    __testDescription: 'Invalid empty object',
    __testIsInvalidSample: true,
    clientId: undefined,
    serviceAccountName: undefined,
```

File: src/components/component/secrets/secret-list-item.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const SecretListItem = ({
 }): JSX.Element => {
   const [isExpanded, setIsExpanded] = useState(false);
 
-  const hasStatusMessages = secret.statusMessages?.length > 0;
+  const hasStatusMessages = secret?.statusMessages?.length > 0;
```
Comment (Pespiri) [inline src/components/component/secrets/secret-list-item.tsx:31]: secret is not an optional input, parent/caller have responsibility to null check
```suggestion
  const hasStatusMessages = secret.statusMessages?.length > 0;
```

File: src/models/azure-identity/test-data.ts [hunk 3]
```diff
@@ -12,23 +12,27 @@ export const testData: TestDependencyDataType<AzureIdentityModel> = [
     __testDescription: 'Valid full object',
     clientId: 'any-client-id',
     serviceAccountName: 'any-sa',
+    azureKeyVaults: ['kv1', 'kv2'],
   },
```
Comment (Pespiri) [inline src/models/azure-identity/test-data.ts:16]: add valid test for partial object
```suggestion
  },
  {
    __testDescription: 'Valid partial object',
    clientId: 'any-client-id',
    serviceAccountName: 'any-sa',
  },
```

**Review by Pespiri — CHANGES_REQUESTED** [review]

**Comment by Pespiri** [comment]
remember to update model for ScheduledJobSummary (src/models/radix-api/deployments/scheduled-job-summary)
image

**Comment by satr** [comment]
> remember to update model for ScheduledJobSummary (src/models/radix-api/deployments/scheduled-job-summary) image
Added field

**Review by Pespiri — APPROVED** [review]

**Pull request merged**